.github/workflows/aws-minimal-schema.yml

```diff
@@ -0,0 +1,38 @@
+name: minimal-schema
+
+description: |
+  Check that minimal schema is up-to-date.
+
+on:
+  pull_request:
```

Comment:
Should we also run that as part of the release process? That would guard against releasing an inconsistent schema (e.g. when mismerges happen)

Reply:
Yes I will follow up. I cannot test this workflow before it merges, I'll need some more PRs to get this right.